packages/@remirror/core/src/builtins/plugins-extension.ts

```diff
@@ -377,15 +377,15 @@ declare global {
        * the plugin state.
        *
        * ```ts
-       * import { ExtensionPluginSpec } from 'remirror/core';
+       * import { CreatePluginReturn } from '@remirror/core';
```

Comment:
Good spot!

Reply:
Although it should be `remirror/core`. It's much easier for users to install `remirror` with everything included than install each package individually. Since it's the recommended way of getting started I've decided for all the core docs to assume that's the way it's being used. Advanced users will decide which they prefer and can probably figure out the difference for themselves.


```suggestion
       * import { CreatePluginReturn } from 'remirror/core';
```